Sprint11

AlexandraRik opened the pull request:
Домашнее задание спринт11

AlexandraRik committed:
11 спринт Домашнее задание
Добавлена библиотека для поиска песен
Приложение делает запрос

AlexandraRik committed:
11 спринт Домашнее задание
Добавлены плейсхолдеры
Темная тема
+кнопка обновления
очищение поиска

IliaAnK commented on app/src/main/java/com/example/playlistmaker/SearchActivity.kt:
```diff
@@ -1,104 +1,173 @@
+        .baseUrl("https://itunes.apple.com")
+        .addConverterFactory(GsonConverterFactory.create())
+        .build()
+    private val songService = retrofit.create(SongApiService::class.java)
```
Рекомендация:
Создание всего ретрофит клиента можно вынести в отдельный файл

AlexandraRik committed:
11 спринт Домашнее задание
Исравлены замечания:
(цвет курсора, вынесение в отдельный файл переменной ретрофит, высвечивание 2 плейсхолдеров подряд)

AlexandraRik committed:
11 спринт Домашнее задание
Убрала комментарии по ретрофит в файле SearchActivity
